softdev.bib

```diff
@@ -1622,3 +1622,13 @@ @article{chen16robust
     biburl    = {http://dblp.uni-trier.de/rec/bib/journals/corr/ChenR16},
     bibsource = {dblp computer science bibliography, http://dblp.org}
 }
+
+@article{antoch97effect,
+    author = {Antoch, Jaromir and Huskova, Marie and Prášková, Zuzana},
```

Comment:
They're equivalent in this case, and we use both styles throughout the file, so I think it's OK. [From memory, I think you have to use "last, first" for some styles of name to render properly. I might be wrong.]